content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -93,6 +109,20 @@ the `startingCSV` field of the subscription is not possible when using skiprange
 The above warning means that skiprange by itself is useful for teams who are not interested in
 supporting directly installing versions within a given range.
 
+#### Example for SkipRange
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+    name: elasticsearch-operator.v4.1.2
+    namespace: placeholder
+    annotations:
+        olm.skipRange: '>=4.1.0 <4.1.2'
+```
+
+In the above example, `elasticsearch-operator.v4.1.2` will replace all operators in its package with versions falling in the range `>=4.1.0` `<4.1.2`, while at the same time preventing those operators from being installed.
+
```

Comment:
Is not the skip range actually removing the project from the index? @anik120

Reply:
Not that skip and skipRange does not work the same way. One of them remove the project from the index and the other not. So, (imo) we need to add this info as well and clarify it.

Reply:
@anik120 @timflannagan @njhale @hasbro17 - Anyone please clarify on the above points.